BrowserKit/Sources/SiteImageView/FaviconURLProcessing/FaviconURLHandler.swift

```diff
@@ -38,6 +38,12 @@ struct DefaultFaviconURLHandler: FaviconURLHandler {
                 await urlCache.cacheURL(cacheKey: model.cacheKey, faviconURL: url)
                 return url
             } catch {
+                let isServerError = ServerErrorHelper.extractServerError(error)
```

Comment:
Could we have unit tests for this? You might have to remove the `ServerErrorHelper` being static to do so